src/main/java/graphql/schema/GraphQLDirective.java

```diff
@@ -93,6 +96,10 @@ public GraphQLArgument getArgument(String name) {
         return null;
     }
 
+    public boolean isExposeViaIntrospection() {
```

Comment:
How about this idea

We have a global boolean which is "show directives in introspection" eg a static

We also have this in case people want to filter certain "secret" directives out of the list

That way the general case is covered - that is with one boolean you can turn on directives in introspection and the specific case is still possible, that is some directives may or maybe not be shown (but with more effort)

Thoughts?